.github/workflows/serverDeployment.yaml

```diff
@@ -0,0 +1,47 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Server Deployment
+
+on: 
+  push:
+    branches:
+      - testServerDeployment
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    # Require that the 'test' job completes first
+    #needs: test
+    #defaults:
+    #  run:
+    #    working-directory: server
+
+    steps:
+    - name: DeployServer
+      uses: appleboy/ssh-action@master
+      with:
+        debug: true
+        host: ${{ secrets.HOST }}
+        username: ${{ secrets.USERNAME }}
+        key: ${{ secrets.SERVERKEY }}
+        port: ${{ secrets.PORT }}
+        script:  |
+          export ENV_FILE=settings/base.conf

```

Comment:
isnt this a duplicate of dockerStart.py?

Reply:
No this is the Github action that Github is running on the the ssh connection to the server.